### PR TITLE
Make 0.8 compile

### PR DESCRIPTION
**Pull request by cemozerr**
## PR Description
Makes all fixes necessary to compile 0.8.

### CODE REVIEW

File: ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/TimingProcessor.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import tech.pegasys.artemis.service.serviceutils.ServiceConfig;
+import tech.pegasys.artemis.storage.ChainStorageClient;
+import tech.pegasys.artemis.util.alogger.ALogger;
+
+import java.util.Date;
+
+import static tech.pegasys.artemis.datastructures.Constants.SECONDS_PER_SLOT;
+import static tech.pegasys.artemis.statetransition.util.ForkChoiceUtil.on_tick;
+
+public class TimingProcessor {
```
Comment (jrhea) [inline ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/TimingProcessor.java:16]: Worst name ever.  I would have gone with...TimeController. 😜

Comment (cemozerr) [reply ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/TimingProcessor.java:16]: oh come on! I thought you'd love it. I'm heartbroken ...

File: ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/TimingProcessor.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  private void onTick(Date date) {
+    try {
+      on_tick(chainStorageClient.getStore(), UnsignedLong.valueOf(date.getTime() / 1000));
+      if (chainStorageClient.getStore().getTime()
+              .compareTo(chainStorageClient.getGenesisTime().plus(nodeSlot.times(UnsignedLong.valueOf(SECONDS_PER_SLOT)))) >= 0) {
+        this.eventBus.post(new SlotEvent(nodeSlot));
+        STDOUT.log(Level.INFO, "******* Slot Event *******", ALogger.Color.WHITE);
+        STDOUT.log(Level.INFO, "Node slot:                             " + nodeSlot);
+        nodeSlot = nodeSlot.plus(UnsignedLong.ONE);
+        Thread.sleep(SECONDS_PER_SLOT * 1000 / 2);
```
Comment (jrhea) [inline ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/TimingProcessor.java:39]: Never understood why we calc this every time.  If slots were 20 seconds...i dont think we would sleep 10 seconds.

**Review by jrhea — APPROVED** [review]
made some smart ass comments.  do what u will. lgtm

File: ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/TimingProcessor.java (hunk 2 above)
Comment (cemozerr) [reply ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/TimingProcessor.java:39]: That's casper commandments for you. bruh.

**Pull request merged**